docker-compose.acra-censor-demo.yml

```diff
@@ -103,9 +103,9 @@ services:
     #===== OWASP Mutillidae II =================================================
 
     edoz90_mutillidae:
-        # Build base image from edoz90/docker-mutillidae
+        # Build base image
         build:
-            context: https://github.com/edoz90/docker-mutillidae.git
+            context: https://github.com/storojs72/docker-mutillidae.git
         image: edoz90_mutillidae:latest
```

Comment:
I'd suggest renaming the image in case of using patched one. For example, it may be `storojs72/edoz90_mutillidae`.

Reply:
agree with @shadinua

Reply:
but it's a fork, probably we can't just rename fork. but we can write some comment

Reply:
Renaming the image we create in docker-compose won't affect the naming of the git repository.